transform/mattermost-analytics/models/staging/mm_telemetry_rc/stg_mm_telemetry_rc__performance_events.sql

```diff
@@ -1,15 +1,9 @@
 {%- set include_columns = ["context_page_referrer", "channel", "context_app_namespace", "user_actual_id"
 , "context_library_name", "type", "context_app_version" , "user_actual_role", "context_page_url", "context_os_name" 
 , "context_page_title", "context_app_build", "context_page_search" , "context_library_version"
-, "context_useragent", "context_app_name", "context_locale", "context_screen_density"  
+, "context_useragent", "context_app_name", "context_locale", "context_screen_density" 
 , "context_page_path", "context_os_version", "category" , "duration" , "num_of_request", "max_api_resource_size"
-, "longest_api_resource_duration" , "user_id", "root_id", "post_id", "sort" 
-, "team_id", "keyword", "count", "gfyid", "field"
-, "plugin_id", "installed_version", "group_constrained", "value", "include_deleted" 
-, "role", "privacy", "scheme_id", "metric", "error", "num_invitations_sent" 
-, "num_invitations", "channel_sidebar", "app", "method", "remaining", "screen" 
-, "filter", "version", "uuid_ts"] 
--%}
+, "longest_api_resource_duration" , "user_id", "count", "uuid_ts"] -%}
```

Comment:
`uuid_ts` can also be removed as it's used internally by Rudderstack.

Reply:
Removed `uuid_ts`